README.md

```diff
@@ -325,6 +326,189 @@ const commonProps = {
 
 A warning is displayed when no fallback image is passed. This warning can be disabled in special circumstances. To disable this warning, look in the [warnings section](#warnings).
 
+#### ImgixProvider Component
+
+The `<ImgixProvider>` Higher Order Component (HOC), makes its [props](#props) available to any nested `<Imgix>` component in your React application.
+
+For example, by rendering `<ImgixProvider>` at the top level of your application with `imgixParams` defined, all your `<Imgix>` components will have access to the same `imgixParams`.
+
+```jsx
+import React from "react";
+import Imgix, { ImgixProvider } from "react-imgix";
+import HomePage from "./components/HomePage";
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <ImgixProvider imgixParams={{ ar: "16:9" }}>
+          <div className="intro-blurb">{/* ... */}</div>
+          <div className="gallery">
+            <Imgix src="https://assets.imgix.net/examples/pione.jpg" />
+            <Imgix src="https://sdk-test.imgix.net/ساندویچ.jpg" />
+          </div>
+        </ImgixProvider>
+      </header>
+    </div>
+  );
+}
+
+export default App;
+```
+
+So that the generated HTML looks something like
+
+```html
+<div class="gallery">
+  <img
+    src="https://assets.imgix.net/examples/pione.jpg?auto=format&ar=16%3"
+    ...
+  />
+  <img
+    src="https://sdk-test.imgix.net/%D8%B3%D8%A7%D9%86%D8%AF%D9%88%DB%8C%DA%86.jpg?auto=format&ar=16%3"
```

Comment:
Looks like you trimmed off the `A9` as well, so this won't give us an encoded `:` anymore. Also, I just noticed that this example ([`8382f4a` (#799)](https://github.com/imgix/react-imgix/pull/799/commits/8382f4a5e4e32d53731deed5369a0effe5bac2e5#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R344)) needs `fit: crop` in order for the `ar` example to work.

```suggestion
    src="https://assets.imgix.net/examples/pione.jpg?auto=format&fit=crop&ar=16%3A9"
    ...
  />
  <img
    src="https://sdk-test.imgix.net/%D8%B3%D8%A7%D9%86%D8%AF%D9%88%DB%8C%DA%86.jpg?auto=format&fit=crop&ar=16%3A9"
```

Reply:
Thanks good cathc. My apols for that botched fix.